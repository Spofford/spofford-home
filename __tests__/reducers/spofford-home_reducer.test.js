import Reducer from '../../app/redux/reducers';

import {
  USER_AUTH, // Only ones related to the reducer being tested
} from '../../app/redux/actions';

describe('USER_AUTH', () => {
  test('returns the correct state', () => {
    const action = { type: USER_AUTH, payload: { user: {email: "amhasler@gmail.com", first_name: "adam", last_name: "hasler", role: "designer", id: 5, charges: []}} };

    expect(Reducer(undefined, action)).toMatchSnapshot();
  });
});
