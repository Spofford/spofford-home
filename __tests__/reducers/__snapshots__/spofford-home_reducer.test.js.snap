// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`USER_AUTH returns the correct state 1`] = `
Object {
  "charge": Object {
    "amount": "",
    "id": "",
    "user_id": "",
  },
  "comment": Object {
    "approved": "",
    "comments": "",
    "created_at": "",
    "id": "",
    "submission_id": "",
    "updated_at": "",
    "user_id": "",
    "version": "",
  },
  "content": Object {
    "model": "",
  },
  "error": Object {
    "message": "",
  },
  "modal": true,
  "mySubmissions": Array [],
  "posts": Object {
    "items": Array [],
  },
  "submission": Object {
    "advance": "",
    "approved": "",
    "cad_url": "",
    "comments": Array [],
    "description": "",
    "id": "",
    "manufacturing": "",
    "photo_url": "",
    "updated_at": "",
  },
  "user": Object {
    "charges": Array [],
    "email": "",
    "first_name": "",
    "id": "",
    "last_name": "",
    "role": "",
  },
}
`;
